docs/modules/ROOT/pages/advanced-how-tos/releasing/tenant-release-pipelines.adoc

```diff
@@ -0,0 +1,79 @@
+= Tenant Release Pipelines
+
+The usual release process in {ProductName} involves two different teams: a *Development team* and a *Managed environment team* as described in
+xref:/advanced-how-tos/releasing/index.adoc[Releasing an application]. But, sometimes the Development team wants to release their software to some destination that is directly under their control, using their own secrets, without depending on a Managed environment team. In {ProductName} we call this model a *tenant release pipeline*. It's a release pipeline that runs in the tenant namespace of the Development team, rather than in that of the Managed environment team.
+
+The gist here is that you are going to follow the regular instructions to xref:/advanced-how-tos/releasing/create-release-plan.adoc[create a ReleasePlan], but you will omit the `target` and specify the `pipelineRef` directly on your `ReleasePlan`.
+
+
+.*Prerequisites*
+
+* You have an existing Development workspace.
+* You have completed the steps listed in the xref:/getting-started/cli.adoc[Getting started in the CLI] page.
+* Either choose one of the link:https://github.com/konflux-ci/release-service-catalog[konflux release pipelines], or write your own.
+
+NOTE: If you write a good reusable release pipeline, please submit it to our link:https://github.com/konflux-ci/release-service-catalog[catalog] so others can use it.
+
+.Procedure
+
+. Create a `ReleasePlan.yaml` object locally.
+
++
+*Example `ReleasePlan.yaml` object*
+
++
+[source,yaml]
+----
+apiVersion: appstudio.redhat.com/v1alpha1
+kind: ReleasePlan
+metadata:
+ labels:
+   release.appstudio.openshift.io/auto-release: 'true' <.>
+   release.appstudio.openshift.io/standing-attribution: 'true'
```

Comment:
Is this still required for just a tenant release pipeline?

Reply:
I checked with @theflockers. This is still required. If it moves to be optional, we can update the docs.

Reply:
The release will fail validation (before any pipelines at all run) if the attribution data is not valid. This is part of that